extras/deepstream/deepstream-sample/ds_config_1vid.txt

```diff
@@ -0,0 +1,73 @@
+# Copyright (c) 2018 NVIDIA Corporation.  All rights reserved.
+#
+# NVIDIA Corporation and its licensors retain all intellectual property
+# and proprietary rights in and to this software, related documentation
+# and any modifications thereto.  Any use, reproduction, disclosure or
+# distribution of this software and related documentation without an express
+# license agreement from NVIDIA Corporation is strictly prohibited.
+
+[application]
+enable-perf-measurement=1
+perf-measurement-interval-sec=1
+
+[tiled-display]
+enable=0
+rows=1
+columns=1
+width=1280
+height=720
```

Comment:
Does this have to match the TRT engine input size? If so we should add the --size to export and mention it in the readme

Reply:
Pretty sure this width and height is referring to the dimensions to use for the output video stream: for other deepstream samples, they also use a similar value, and their network input size does not match either.